crates/swift-bridge-ir/src/codegen/codegen_tests/result_codegen_tests.rs

```diff
@@ -628,12 +638,15 @@ mod extern_rust_fn_return_result_tuple_type_and_transparent_enum_type {
         }
     }
 
+    // Allows unused to avoid dead_code warnings in Rust 1.79.0 or later.
```

Comment:
Let's:

- link to the Rust issue
- make it clear that this can be deleted when that issue is closed

This way no matter where you run into this `#[allow(unused)` you can easily check whether it can be removed.